libexec/bats-exec-test

```diff
@@ -68,6 +71,10 @@ load() {
   source "${filename}"
 }
 
+# timeout from here: http://stackoverflow.com/questions/3504945/timeout-command-on-mac-os-x
+# in case 'timeout' command does not exist on the system
+function timeout() { perl -e 'alarm shift; exec @ARGV' "$@"; }
```

Comment:
The drawback with launching perl is that it is expensive.  You might want to use timeout command when available, and then fall back to perl.  If, the goal is to get this into the master BATS project at some point they will want a pure BASH method.